handlers/main.yml

```diff
@@ -2,12 +2,12 @@
 
 - name: Restart gdm
   service:
-    name: 'gdm3'
+    name: 'display-manager'
```

Comment:
I am not sure anymore if there was a specific reason I made this explicit instead of using `display-manager` but I am open to try it. But your patch is not complete. Can you drop of the handler tasks now as they are redundant now. Also, the name should reflect what the task does.

Reply:
I have unified the handlers.